app/utils/nonsipp/check/SharesCheckStatusUtils.scala

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.nonsipp.check
+
+import models.ConditionalYesNo._
+import pages.nonsipp.shares._
+import models.IdentityType._
+import config.RefinedTypes.{Max5000, OneTo5000}
+import eu.timepit.refined.refineV
+import models._
+import pages.nonsipp.common._
+import models.IdentitySubject._
+import models.TypeOfShares.{ConnectedParty, SponsoringEmployer, Unquoted}
+import models.SchemeId.Srn
+
+object SharesCheckStatusUtils {
```

Comment:
No changes were made to this code, I just copied the Shares related code from CheckStatusUtils and made this its own file.